### PR TITLE
fixes "you mouth dry" [we're the top server now]

### PR DESCRIPTION
**Pull request by JohnOxford**
fixes shitty slav english

fighting communism one pr at a time

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -685,25 +685,25 @@
 	//THIRST//
 	if(H.water > THIRST_LEVEL_LIGHT)
 		if(H.transpiration_efficiency != 1.1)
-			H << "<span class='notice'>You quench your thirst.</span>"
+			H << "<span class='notice'>You are no longer thirsty.</span>"
 		H.transpiration_efficiency = 1.1
 	else if(H.water > THIRST_LEVEL_MIDDLE) //LITLE THIRST
 		if(H.transpiration_efficiency != 1)
-			H << "<span class='notice'>You mouth dry.</span>"
```
Comment (Angryon) [inline code/modules/mob/living/carbon/human/species.dm]: you mouth dry is non-negotiable

Comment (JohnOxford) [reply code/modules/mob/living/carbon/human/species.dm]: fuggin heresay

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -918,13 +918,13 @@
...
 				else
-					target << "<span class='warning'>Your last hair is gone</span>"
+					target << "<span class='warning'>Your hair is completely gone.</span>"
 					spawn(50)
 						target.facial_hair_style = "Shaved"
 						target.hair_style = "Bald"
 						target.update_hair()
 		if (totalRads>=50&&prob(2))
-			target << "<span class='danger'>You feel different...</span>"
+			target << "<span class='danger'>You feel different..</span>"
```
Comment (AndrewL97) [inline code/modules/mob/living/carbon/human/species.dm]: ellipses uses three periods, keep this as ...

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -685,25 +685,25 @@
...
 		H.transpiration_efficiency = 1
 	else if(H.water > THIRST_LEVEL_HARD) //MIDDLE THIRST
 		if(H.transpiration_efficiency != 0.9)
-			H << "<span class='warning'>Mouth completely dry, you feel weak.</span>"
+			H << "<span class='warning'>You are thirsty.</span>"
 		H.transpiration_efficiency = 0.9
 	else if(H.water > THIRST_LEVEL_DEADLY) //HARD THIRST
 		if(H.transpiration_efficiency != 0.6)
-			H << "<span class='warning'>Thirst is slowly killing you.</span>"
+			H << "<span class='warning'>You are dehydrated.</span>"
```
Comment (AndrewL97) [inline code/modules/mob/living/carbon/human/species.dm]: players usually need the words "dying" or "killing" to actually tell them that they are about to die. This message usually comes up just before they start to collapse and die, i'd prefer these stay as urgent as they are

File: code/modules/mob/living/carbon/human/species.dm [hunk 4]
```diff
@@ -685,25 +685,25 @@
 	//THIRST//
 	if(H.water > THIRST_LEVEL_LIGHT)
 		if(H.transpiration_efficiency != 1.1)
-			H << "<span class='notice'>You quench your thirst.</span>"
+			H << "<span class='notice'>You are no longer thirsty.</span>"
 		H.transpiration_efficiency = 1.1
 	else if(H.water > THIRST_LEVEL_MIDDLE) //LITLE THIRST
 		if(H.transpiration_efficiency != 1)
-			H << "<span class='notice'>You mouth dry.</span>"
+			H << "<span class='notice'>YOU MOUTH DRY</span>"
```
Comment (AndrewL97) [inline code/modules/mob/living/carbon/human/species.dm]: pls revert, still says mouth dry but the original message is now tradition

File: code/modules/mob/living/carbon/human/species.dm [hunk 5]
```diff
@@ -670,7 +670,7 @@
 
 	else if(H.nutrition < NUTRITION_LEVEL_STARVING - 100)
 		if(H.metabolism_efficiency != 0.6)
-			H << "<span class='warning'>You are deadly hungy.</span>"
+			H << "<span class='warning'>You are starving to death.</span>"
```
Comment (AndrewL97) [inline code/modules/mob/living/carbon/human/species.dm]: This is the urgency I like. Keep this one

**Pull request closed without merging**